chapters/substitution.adoc

```diff
@@ -2,13 +2,25 @@
 .Definition
 Robots are substituted by the <<Robot Handler, robot handler>> of the respective team. No other team member is allowed to take robots out or put robots in.
 
-. A <<Robot Handler, robot handler>> who's team is marked for robot substitution performs at least one of the following operations in the given order.
-.. Put robots in. The maximum allowed number of robots of the team on the field must not be exceeded.
-.. Take robots out. If a robot that is taken out is closer than 1 meter to the intersection of the halfway line with one of the touch lines, another robot can be put in right away. Otherwise the robot handler has to wait until the next substitution opportunity.
+. The <<Robot Handler, robot handler>> should prefer to use long sleeves and colors that won't interfere with the vision system.
+
+. Robots can always be substituted during game play if the following specifications are respected:
+
+.. The robot is at least partially inside the <<Field Surface, field margin>>
+.. The robot is at a distance from the halfway line that must not exceed 1 meter
+.. The ball must be at least 2 meters away from the halfway line
+
+. If the previous specifications are respected, the <<Robot Handler, robot handler>> is allowed to put robots in and take robots out, without notifying the <<Referee, referee>>.
```

Comment:
Sry for nitpicking, but this is sentence seems partially redundant. You already said before, that the specifications have to be respected and now you repeat it. I would just add the "without notifying to the point above:

> Robots can always be substituted during game play without notifying the <<Referee, referee>> if the following specifications are respected:

Reply:
No problem, if you find anything that could be improved just tell me